etc/google-play-music-desktop-player.profile

```diff
@@ -0,0 +1,16 @@
+# Google Play Music desktop player profile
+noblacklist ~/.config/Google Play Music Desktop Player
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-programs.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-passwdmgr.inc
+
+caps.drop all
+seccomp
+protocol unix,inet,inet6,netlink
+noroot
+
+#whitelist ~/.pulse
+#whitelist ~/.config/pulse
```

Comment:
Did you mean to comment out these lines?
Also, does this program go online? If so, you might want to consider adding `netfilter`.